examples/11_particle_dataframe.py

```diff
@@ -28,8 +35,22 @@
 df = electrons.to_df(np.s_[:100])
 print(df)
 
-# get optimal chunks
-for chunk in electrons["position"]["x"].available_chunks():
-    stride = np.s_[chunk.offset[0]:chunk.extent[0]]
-    df = electrons.to_df()
+
+# Dask example: process large datasets
+if found_dask:
+    df = electrons.to_dask()
     print(df)
+
+    # trigger a couple of compute actions
+    #   example1: average momentum in z
+    print("<momentum_z>={}".format(df["momentum_z"].mean().compute()))
+
+    #   example2: momentum histogram
```

Comment:
2D histograms in dask for a phase space example:
- https://github.com/dask/dask/pull/7346
- https://github.com/xgcm/xhistogram/